android/src/main/kotlin/com/fluttercandies/photo_manager/core/utils/IDBUtils.kt

```diff
@@ -428,11 +429,6 @@ interface IDBUtils {
         }
         refreshStream()
 
-        val shouldKeepPath = if (!isAboveAndroidQ) {
```

Comment:
Could you also keep the condition? The fix should work without removing it, correct?